Training/configs/training_v1.yaml

```diff
@@ -267,7 +267,7 @@ Features_all :
                 - pfCand_ele_valid: [*is_pfElectron, null, "no_scaling"]
                 - pfCand_ele_rel_pt: [*is_pfElectron, {"pfCand_ele_rel_pt": "pfCand_pt/tau_pt"}, "normal", -5, 5]
                 - pfCand_ele_deta: [*is_pfElectron, {"pfCand_ele_deta": "pfCand_eta - tau_eta"}, "linear", {'inner': [-0.1, 0.1], 'outer': [-0.5, 0.5]}]
-                - pfCand_ele_dphi: [*is_pfElectron, {"pfCand_ele_dphi": "pfCand_phi - tau_eta"}, "linear", {'inner': [-0.1, 0.1], 'outer': [-0.5, 0.5]}] # NB: will not bring dphi values to [-pi,pi] on the fly here
+                - pfCand_ele_dphi: [*is_pfElectron, {"pfCand_ele_dphi": "pfCand_phi - tau_phi"}, "linear", {'inner': [-0.1, 0.1], 'outer': [-0.5, 0.5]}] # NB: will not bring dphi values to [-pi,pi] on the fly here
```

Comment:
- please fix similar typos in all other places
- this fix should affect the scaling output because `dphi` is always scaled to the fixed interval, so the current version of `scaling_params_v2.json` should be correct.
- path to the spectrum files should also be changed in `training_v1.yaml` (and the spectrum files should be updated to the latest version). Please add this step to the instructions in the readme.